policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml

```diff
@@ -1,5 +1,6 @@
 description: Virginia allows a full age deduction for taxpayers born before this year.
 values:
+  2021-01-01: 1957
```

Comment:
just change to 2021-01-01: 1939

Reply:
but please double check the structure to make sure that we calculate it correctly

Reply:
I have double checked the structure and the value should be 1939

Reply:
can you change it?